react/src/components/PayButton/PayButton.stories.tsx

```diff
@@ -63,13 +68,13 @@ withUSDGoalCurrency.storyName = 'With USD Goal';
 withUSDGoalCurrency.args = {
   animation: 'none',
   goalAmount: 500000,
-  currency: 'USD',
+  currency: defaultCurrency,
```

Comment:
This isn't right: `defaultCurrency` here will evaluate to either `'XEC'` or `'BCH'` (in this case to `'XEC'`,  since `defaultArgs.to` is an `ecash:` address), but here it should be `'USD'`. 

I think you can leave the hardcoded 'USD' string since  the story is specific about it (`withUSDGoalCurrency`)